pkgs/tools/security/vault/vault-bin.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "vault-bin-${version}";
+  version = "1.1.3";
+
+  src = fetchurl {
+    url = "https://releases.hashicorp.com/vault/${version}/vault_${version}_linux_amd64.zip";
+    sha256 = "293b88f4d31f6bcdcc8b508eccb7b856a0423270adebfa0f52f04144c5a22ae0";
+  };
+
+  buildInputs = [ unzip ];
+
+  buildCommand = ''
+    mkdir -p $out/bin $out/share/bash-completion/completions
+    unzip $src
+    mv vault $out/bin
+    echo "complete -C $out/bin/vault vault" > $out/share/bash-completion/completions/vault
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://www.vaultproject.io;
+    description = "A tool for managing secrets, this binary includes the UI";
+    platforms = [ "x86_64-linux" ];
+    license = licenses.mpl20;
+    maintainers = with maintainers; [ rushmorem lnl7 offline pradeepchhetri psyanticy ];
```

Comment:
Can you please remove other people from maintainers, as I don't know if they want to be included. Leave yourself and you can leave me.

Reply:
Done